frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -81,6 +81,30 @@ describe("scenarios > dashboard", () => {
     cy.findByText("How many orders were placed in each year?");
   });
 
+  it("should allow empty card title (metabase#12013)", () => {
+    cy.intercept("GET", "/api/table/**/query_metadata").as("tableMetadata");
+    visitDashboard(1);
+
+    // 3 tables
+    cy.wait("@tableMetadata");
+    cy.wait("@tableMetadata");
+    cy.wait("@tableMetadata");
```

Comment:
`visitDashboard` helper already waits for the `dashcardQuery` which is happening **after** tableMetadata loaded. Are you sure you need this?
<img width="598" alt="image" src="https://user-images.githubusercontent.com/31325167/165861774-3dec54e6-d932-48e6-909b-9addb660b894.png">

Reply:
Good point! I was unnecessarily cautious, I will remove those redundant checks.